modyn/selector/selector.py

```diff
@@ -112,14 +112,11 @@ def uses_weights(self) -> bool:
         return self._strategy.uses_weights
 
     def get_selection_strategy_remote(self) -> tuple[bool, str, dict]:
-        assert not (
-            self._strategy._requires_remote_computation and not isinstance(self._strategy, AbstractDownsampleStrategy)
-        )
-
-        if self._strategy._requires_remote_computation and isinstance(self._strategy, AbstractDownsampleStrategy):
+        if isinstance(self._strategy, CoresetStrategy):
+            requires_remote_computations = self._strategy.get_requires_remote_computation()
```

Comment:
small typo (no s to match the function call) - but we might want to change this again to not use this "hack"